### DISCUSSION TITLE
Map collection/pack file type

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18682] [Ideas]
### Big idea alert:

- A new file type, `.osc` or something, which is a list of beatmap links/ID's. 
- you could export a collection to this file, and
- people would be able to import this list into their game, and maybe download ones they choose or download the entire thing? either way,
- it would allow people to share groups of maps much easier, like, "180 bpm stream pack" and if you downloaded and imported into your game, you could have all of them in a new collection? or just downloaded into your main collection.

**Comment by peppy** [comment]
Rather than using files, you can use the playlists system to roughly achieve this already.

**Reply by Axilotl17** [reply]
the issue is that you cant just... send a playlist to someone as easily, and they expire - im talking a mapset with multiple songs people can save and send for maybe years. I think playlists while having a very cool leaderboard feature, don't function as this could if done right, where you can multi-download maps maybe already in a collection.
